docs/src/tut/mkt/index.md

```diff
@@ -0,0 +1,1323 @@
+
+* Lines 9 to 12 execute the `Seller`'s contract on the `Seller`. 
+The `Seller` is also given access to `commonInteract` and `sellerInteract` properties. 
+
+* Lines 13 to 16 execute the `Buyer`'s contract on the `Buyer`. 
+But this time around, the `Buyer` is given access to the `commonInteract` and `buyerInteract` properties.
+
+The code that you have just written forms the basis for the DApp that is to be built.
+
+Notice that `commonInteract`, `buyerInteract`, and `sellerInteract` are similar in both the backend and frontend files. 
+
+This is so because `Reach` needs to be sure of how the variables are to be used before being used. 
+The program will not be executed if those variables are not used the way they were declared.
+So these variables are declared at the backend before being defined in the frontend. 
+
+This might feel like a lot of work especially if you are coming from a language like JavaScript that is weakly typed.
+But we will rather use an application with strong security than one with weak security especially when it has to do with money and trust.
+
+
+### {#tut-mkt-starter-Review} Review
+
+In this section, we created the boilerplate for the application that this tutorial plans to build.
+
+Two (2) important files were created. And the necessary code was added. 
+See full code [here]().
+
+Next, a simple DApp will be built. This will help to visualize what the finished product will look and feel like.
+If the diagram above wasn't clear enough, this next section will help.
+
+Time to proceed!
+
+
+
+## {#tut-mkt-basic} Basic DApp
+
+In this section, a basic DApp will be created to demonstrate what the finished product will look like. 
+By the end, you will see what a basic Reach command-line DApp works.
+
+### {#tut-mkt-basic-why} Why Build This Basic DApp
+A `seller` relocates to an area where there is high demand for the goods he sells. 
+He enjoys a monopoly in the market and soon realizes that he has more `buyer`s than he can handle.
+He also finds out that the method he has been using to transact business isn't secure enough as the business grows.
+
+He then decides to build an application that will make it easy for him to transact business with little or no supervision. 
+Since he is new to this technology and enjoys a monopoly, he decides to build a simple DApp that gives no autonomy to him or the `buyer`. 
+Once he and a `buyer` agree to transact business, the DApp picks a good that both parties must agree to transact upon.
+
+You are contracted to build this application and you follow these steps to successfully build the application:
+
+
+### {#tut-mkt-basic-declare} Declare Variables
+
+Enter the following code just before the  `commonInteract` `{!rsh} Object`:
+
+```reach
+load: /examples/mkt-2-basic/index.rsh
+range: 3 - 11
+```
+
+These are just constants that will help in building the `commonInteract`, `sellerInteract` and `buyerInteract` `{!rsh} Object`. Nothing surprising.
```

Comment:
These are constants...
 `{!rsh} Object`s

also, "Nothing surprising" is unnecessary